src/Options/Definitions.js

```diff
@@ -533,6 +539,12 @@ module.exports.ParseServerOptions = {
     help: 'Starts the liveQuery server',
     action: parsers.booleanParser,
   },
+  transformEmailAndUsernameToLowerCase: {
+    env: 'PARSE_SERVER_TRANSFORM_EMAIL_AND_USERNAME_TO_LOWER_CASE',
+    help:
+      'Transform Username and Email to lowercase on create/update/login/signup. On queries client needs to ensure to send Email/Username in lowercase format.',
```

Comment:
Is there any way a client can write an email or username non-lowercase, or do all possible routes have the transformation in place?

Reply:
The implementation is at DB controller level, so every create/update are going through those functions.